File: joueur/Level.java
package joueur;

public class Level {

	public static void levelUpdate(Joueur joueur) {
		while (joueur.getExperience() >= joueur.getExperienceRequis()) {
			joueur.setLevel(joueur.getLevel() + 1);
			joueur.setExperience(joueur.getExperience() - joueur.getExperienceRequis());
			joueur.setExperienceRequis(joueur.getLevel());
			System.out.println("Bravo vous etes passez lv " + joueur.getLevel() + " ,prochain level dans " +joueur.getExperienceRequis() + "exp.");
			}
		}

	}

File: Cs/joueur/Level.cs
using System;

namespace joueur
{
    public class Level
    {
        public static void LevelUpdate(Joueur joueur)
        {
            while (joueur.GetExperience() >= joueur.GetExperienceRequis())
            {
                joueur.SetLevel(joueur.GetLevel() + 1);
                joueur.SetExperience(joueur.GetExperience() - joueur.GetExperienceRequis());
                joueur.SetExperienceRequis(joueur.GetLevel());
            }
            Console.WriteLine($"Bravo vous êtes passé lv {joueur.GetLevel()}, prochain level dans {joueur.GetExperienceRequis()} exp.");
        }
    }
}
